src/style.scss

```diff
@@ -98,47 +114,54 @@ body {
       }
     }
     div.details {
```

Comment:
Changed to a nested combo of a flex of rows and block within the row (value over label).
Moved common props out to parent div or span in most cases.
Scale fonts slightly, keeping a max set.
Max of 640 is to keep a pleasing set of 4x or 2x flex units so they don't scale too far apart, but let them get as tight as they need to.